tests/test_cli.py

```diff
@@ -737,7 +737,9 @@ async def test_pki_enrollment(tmp_path, mocked_parsec_ext_smartcard, backend, al
         async def _cli_invoke_in_thread(cmd: str):
             # We must run the command from another thread given it will create it own trio loop
             with trio.fail_after(1):
-                return await trio.to_thread.run_sync(runner.invoke, cli, cmd)
+                return await trio.to_thread.run_sync(
+                    lambda: runner.invoke(cli, cmd, env={"DEBUG": "1"})
```

Comment:
good idea, maybe add a comment `# pass DEBUG env var for better output on crash`

Reply:
4a1617256